webhook-notifications_5def901604286364bc92af37.md

```diff
@@ -0,0 +1,283 @@
+        "name": "semaphore",
+        "id": "36360e31-fee6-42b2-9f6c-999d4c06ce81"
+    },
+    "project": {
+        "name": "notifications",
+        "id": "91e34570-bebe-42b6-b47a-ca710b2b8927"
+    },
+    "repository": {
+        "url": "https://github.com/renderedtext/notifications",
+        "slug": "renderedtext/notifications"
+    },
+    "revision": {
+        "tag": null,
+        "sender": {
+            "login": "radwo",
+            "email": "184065+radwo@users.noreply.github.com",
+            "avatar_url": "https://avatars2.githubusercontent.com/u/184065?v=4"
+        },
+        "reference_type": "branch",
+        "reference": "refs/heads/rw/webhook_impl",
+        "pull_request": null,
+        "commit_sha": "2d9f5fcec1ca7c68fa7bd44dd58ec4ff65814563",
+        "commit_message": "empty",
+        "branch": {
+            "name": "rw/webhook_impl",
+            "commit_range": "36ebdf6e906cf3491391442d2f779b512ca49485...2d9f5fcec1ca7c68fa7bd44dd58ec4ff65814563"
+        }
+    },
+    "workflow": {
+        "initial_pipeline_id": "fa02c7bd-7a8b-42e0-8d6e-aa0d8a194e19",
+        "id": "acabe58e-4bcc-4d39-be06-e98d71917703",
+        "created_at": "2019-12-10T13:09:54Z"
+    },
+    "pipeline": {
+        "yaml_file_name": "semaphore.yml",
+        "working_directory": ".semaphore",
+        "stopping_at": "2019-12-10T13:10:22Z",
+        "state": "done",
+        "running_at": "2019-12-10T13:09:58Z",
+        "result_reason": "user",
+        "result": "stopped",
+        "queuing_at": "2019-12-10T13:09:55Z",
+        "pending_at": "2019-12-10T13:09:55Z",
+        "name": "Notificaitons",
+        "id": "fa02c7bd-7a8b-42e0-8d6e-aa0d8a194e19",
+        "error_description": "",
+        "done_at": "2019-12-10T13:10:28Z",
+        "created_at": "2019-12-10T13:09:54Z"
+    },
+    "blocks": [
+        {
+            "state": "done",
+            "result_reason": "user",
+            "result": "stopped",
+            "name": "List & Test & Build",
+            "jobs": [
+                {
+                    "status": "finished",
+                    "result": "stopped",
+                    "name": "Test",
+                    "index": 1,
+                    "id": "21df03d2-c4e0-4e0a-acd7-5ff60dc0727e"
+                },
+                {
+                    "status": "finished",
+                    "result": "stopped",
+                    "name": "Build",
+                    "index": 2,
+                    "id": "84190263-362c-4051-8260-e43637f148de"
+                },
+                {
+                    "status": "finished",
+                    "result": "passed",
+                    "name": "Lint",
+                    "index": 0,
+                    "id": "d4b93a5b-69a5-43e6-ab24-06b095fc49bf"
+                }
+            ]
+        }
+    ]
+}
+```
+
+In this example `revision.pull_request` and `revision.tag` are null because
+payload is related to pipeline from branch build. Information about build type
```

Comment:
"build" is a deprecated concept from Semaphore Classic. Refer to executed pipelines as pipeline runs.